.github/instructions/dsc-community-style-guidelines-powershell.instructions.md

```diff
@@ -151,7 +151,7 @@ function Get-Something
 - Use `PSCredential` for credentials
 - Avoid hardcoded computer names, use cross-platform [`Get-ComputerName`](https://github.com/dsccommunity/DscResource.Common/wiki/Get%E2%80%91ComputerName) instead of `$env:COMPUTERNAME`
 - Place `$null` on left side of comparisons
-- Avoid empty catch blocks (use `-ErrorAction SilentlyContinue`)
+- Avoid empty catch blocks (use `-ErrorAction Ignore`)
```

Comment:
Switch to `SilentlyContinue` due to there seems be edge cases where it can't be used.